packages/fluentui/react-builder/src/components/Designer.tsx

```diff
@@ -379,6 +382,8 @@ export const Designer: React.FunctionComponent = () => {
     }
   }, []);
 
+  const handleCanvasMessage = (canvasMessage: string) => setCanvasMessage(canvasMessage);
```

Comment:
You should use `useCallback` with empty dependencies array to have referentially stable callback between rerenders.